compiler/rustc_typeck/src/check/wfcheck.rs

```diff
@@ -1298,12 +1298,22 @@ fn check_variances_for_type_defn<'tcx>(
 
         match param.name {
             hir::ParamName::Error => {}
-            _ => report_bivariance(tcx, param.span, param.name.ident().name),
+            _ => {
```

Comment:
sorry for being pedantic, but can you please move this whole branch into `report_bivariance` and change it's argument to `hir::GenericParam` :sweat_smile: 

afterwards r=me

Reply:
nah it's def cleaner this way, was just being a bit lazy